docs/explanation/lm-pipeline.md

```diff
@@ -1,28 +1,28 @@
-# Overview
+# The Language Modeling Pipeline
```

Comment:
This is still not exactly what we want (I guess our framing has shifted with tootsie since the early days of Marin), but this is okay for now.